sdk/search/azure-search-documents/setup.py

```diff
@@ -63,7 +62,7 @@
             "azure.search",
         ]
     ),
-    python_requires=">=3.7",
+    python_requires=">=3.8",
```

Comment:
Should also update the README.md and samples/README.md with the minimum Python bump, and mentioning in the Changelog that Python 3.7 is no longer supported (though this should probably be in the `11.5.0` changelog).